Defs/Stats/Stats_Apparel.xml

```diff
@@ -32,6 +32,30 @@
       </li>
     </parts>
   </StatDef>
+
+  <StatDef>
+    <defName>ApparelNightVisionEfficiency</defName>	  
+    <label>apparel night vision efficiency</label>    
+    <description>Blah Blah Blah</description>
+    <category>Apparel</category>
+    <minValue>0</minValue>
+    <maxValue>1.0</maxValue>    
+    <hideAtValue>0</hideAtValue>    
+    <defaultBaseValue>0</defaultBaseValue>
+    <toStringStyle>PercentZero</toStringStyle>
+    <showIfUndefined>false</showIfUndefined>   
+    <parts>
+      <li Class="StatPart_Quality">
```

Comment:
I'd rebalance this, such that normal = 1 and go from there.

Reply:
Nope, this is intended to force the player to craft goggles instead of looting them.